__tests__/app.test.js

```diff
@@ -0,0 +1,80 @@
+const app = require('../app.js');
+const data = require('../db/data/test-data/index.js');
+const seed = require('../db/seeds/seed.js');
+const db = require('../db/connection.js');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(data);
+});
+
+afterAll(() => {
+    db.end();
+});
+
+describe('GET /api/topics', () => {
+    it('should return a 200 http status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200);
+    });
+
+    test('response data should be on a "topics" key', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty('topics');
+        });
+    });
+    
+    test('response data should contain three topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.topics).toHaveLength(3);
+        });
+    });
+
+    test('each topic should have correct object layout', () => {
+        const objectLayout = {
+            slug: expect.any(String),
+            description: expect.any(String)
+        };
+
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            for (const topic of body.topics) {
+                expect(topic).toMatchObject(objectLayout);
+            }
+        });
+    });
+
```

Comment:
The above tests could be combined into one - I guess you've got some repeated code here (expecting 200, get path) but that's more of a personal choice. If you prefer splitting this out, that's fine!